modules/programs/fish.nix

```diff
@@ -138,9 +138,72 @@ let
     };
   };
 
-  abbrsStr = concatStringsSep "\n"
-    (mapAttrsToList (k: v: "abbr --add --global -- ${k} ${escapeShellArg v}")
```

Comment:
Why are we removing `--global`? Is that not supported any more, I can't find it in the docs?

Reply:
Introduced in https://github.com/nix-community/home-manager/pull/548

Reply:
Looks like this is the [default behaviour](https://github.com/fish-shell/fish-shell/blob/c7b43b3abfb24af324100667932b168f0ef566e3/fish-rust/src/builtins/abbr.rs#L545-L547) now, so should be safe to remove.

Reply:
Oops, you said that in the OP.